conformance/results/pyrefly/protocols_explicit.toml

```
@@ -0,0 +1,16 @@
+conformant = "Fail"
```

Maybe change this from "Fail" to "Unsupported" for consistency?